home/templates/home/home_page.html

```diff
@@ -33,7 +33,15 @@
 <div class="container">
     {% for block in page.featured_articles %}
         <div class="row mb-md-4">
-            {% if block.block_type == "one_column" %}
+            {% if block.block_type == "marquee_banner" %}
+                {% if block.value.banner_type == "moves" %}
+                    <div class="w-100">
+                        <marquee onmouseover="this.stop();" onmouseout="this.start();">
```

Comment:
We should probably break the calls to `this.` into it's own function, this should work, but I don't like the style

Reply:
Function for stopping on the marquee on hover? Using the onmouseover tag seems like the simplest way to do it.